editors/code/package.json

```diff
@@ -312,6 +312,11 @@
                 "command": "rust-analyzer.joinLines",
                 "key": "ctrl+shift+j",
                 "when": "editorTextFocus && editorLangId == rust"
+            },
+            {
+                "command": "rust-analyzer.onBackspace",
+                "key": "backspace",
+                "when": "config.rust-analyzer.enableBackspaceListener && editorTextFocus && editorLangId == rust"
```

Comment:
Wow, so we could have made `onEnter` configurable? :exploding_head:.

Reply:
maybe 😄